client/lib/react-helpers/index.js

```diff
@@ -9,15 +9,25 @@ import { Provider as ReduxProvider } from 'react-redux';
  * Internal dependencies
  */
 import analytics from 'lib/analytics';
+import Stylizer, { insertCss } from './stylizer';
 
 export function concatTitle( ...parts ) {
 	return parts.join( ' › ' );
 }
 
-export function renderPage( context, component ) {
-	renderWithReduxStore(
-		component,
-		document.getElementById( 'primary' ),
+export function renderPage( component, domContainer, context ) {
```

Comment:
@gziolo, @tug: maybe we should make that improvement in another pull request? This one is already quite long.